src/middlewares/task.middleware.js

```diff
@@ -195,16 +195,18 @@ export default (store: any) => (next: any) => (action: any) => {
     }
 
     case COMPLETE_TASK: {
-      const { task } = action;
+      const { task, wasSuccessful } = action;
 
       // Send a message to add info to the terminal about the task being done.
       // TODO: ASCII fish art?
 
-      const message = 'Task completed';
+      /* eslint-disable no-useless-concat */
+      const message = wasSuccessful
+        ? '\u001b[32;1m' + 'Task Completed' + '\u001b[0m'
+        : '\u001b[31;1m' + 'Task Failed' + '\u001b[0m';
```

Comment:
Perhaps use [`chalk`](https://github.com/chalk/chalk) to make this clearer and simpler?

Reply:
That's a brilliant idea! For some reason I didn't think terminal-styling solutions would work, since this is ultimately displayed on the web... but that's why we have `ansi-to-html`.

Reply:
Ah. so, I tried it, and it doesn't appear to work. `chalk.red('Hello')` just outputs a plain string, "Hello". No colors.

`chalk` uses a lower-level package, `ansi-styles`, and _this_ works, but it's really verbose, and it's not clear to me how to use a color plus bold.

So I think we can probably leave this like this, at least for now. Eventually we should find or write our own little util to make this nicer, but it doesn't strike me as mission-critical.

Reply:
Try setting `chalk.level` to `3`. This will [force coloring to be enabled](https://github.com/chalk/chalk#chalklevel). Chalk automatically disables colors when `process.stdout` is not a TTY (i.e. it’s a file or something else that usually doesn’t support colors).

Reply:
Ahh, awesome! I saw the docs for `level`, but it wasn't clear to me that it'd help. Thanks for the tip!